docs/Manage-Space.rst

```diff
@@ -227,8 +227,9 @@ follows:
 
 1. Click |image22| at the top left corner of the **Space** page to open the Add New Space form.
 
-2. In the Settings tab, enter a space name (required and from 3 to 200 characters) 
-   and description (from 0 to 255 characters).
+2. A popup is displayed containing all information related to the space creation.
```

Comment:
```suggestion
2. A popup is displayed containing all informations needed to create the space.
```

Reply:
@plamarque the word information is uncountable and should never take an "s" in the end